code/datums/diseases/_disease.dm

```diff
@@ -79,7 +79,7 @@ GLOBAL_LIST_INIT(diseases, subtypesof(/datum/disease))
 	var/cure = has_cure()
 
 	if(carrier && !cure)
-		return TRUE
+		return FALSE
```

Comment:
Why not just make this `if(carrier)`, if carriers will now not be impacted by the virus at all?

Reply:
We still want carriers to be curable. Else the part of the code that handles the curing will be unreachable for carriers.

Reply:
Ah right, have to cure them so they aren't carriers anymore. Makes sense to me!